lib/membrane_rtc_engine/track_synchronizer.ex

```diff
@@ -0,0 +1,90 @@
+defmodule Membrane.RTC.Engine.TrackSynchronizer do
+  @moduledoc """
+  TrackSynchronizer is responsible for synchronizing audio and video tracks.
+  Synchronizer will drop all of the buffers until both pads are added. Also it will change timestamps accordingly.
```

Comment:
```suggestion
  Element responsible for synchronizing audio and video tracks.
  It will drop all of the incoming buffers until both pads are added and change timestamps accordingly.
```